### ISSUE TITLE
How to set a color scheme/theme?

### ISSUE THREAD

**Issue by edouard-lopez** [OmarTawfik/terminal-screenshot#8]
related: #11 

Hello, thanks for your time invested in the project! 
I'm using it to generate screenshots of the various features in [pure.fish prompt](https://github.com/pure-fish/pure/pull/345/files#diff-00f9ccef527bdc4edb90bcf9281403e63d488530e575b61afe616c91d06eb1abR12-R27) documentation.

I would like to take screenshot using a light theme like [ayu](https://github.com/ayu-theme) or a [base16](https://github.com/tinted-theming/base16-shell). How would one do that?

**Comment by edouard-lopez** [comment]
Looking around I fund the https://github.com/ysk2014/xterm-theme project that exposes [theme's object](https://github.com/ysk2014/xterm-theme/blob/master/src/iterm/ayu.js) and allow to do:

```
import { Terminal } from 'xterm';
import { ayu } from 'xterm-theme';

import xtermTheme from 'xterm-theme';

const terminal = new Terminal({
    theme: ayu
});
```

**Comment by OmarTawfik** [comment]
Good idea. Maybe we can add an `options.theme` to replace `options.backgroundColor` .. I suggest accepting the xterm's `theme` object ([link](https://xtermjs.org/docs/api/terminal/interfaces/iterminaloptions/#optional-theme)). I'm a bit skeptic about depending on the `xterm-theme` package. It is a 5 years old package with no real usage or known contributors. But the suggestion above would permit users of `terminal-screenshot` to leverage `xterm-theme` if they want, and forward its data along.
